lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -58,7 +58,7 @@ function _campaign_resource_definition() {
             'name' => 'count',
             'description' => 'The number of campaigns to retrieve.',
             'optional' => TRUE,
-            'type' => 'string',
+            'type' => 'int',
```

Comment:
This seems fine. Just curious what it was before that broke when it was a  `string` and if there's potential for anything new to break with this change.